.github/workflows/documentation-tests.yaml

```diff
@@ -0,0 +1,106 @@
+name: Documentation build and test
+on:
+  workflow_dispatch:
+    inputs:
+      optional_documentation_branch:
+        description: 'Optional Documentation branch name'
+        required: false
+        type: string
+
+  push:
+    branches:
+      - 'master'
```

Comment:
Should we use docs `2.6.x` branch here instead of master? If accepted, update this entire file

Reply:
I think it is still not applied, neither in line 20 of this same workflow

Reply:
7f5610dfb